content/webapp/services/prismic/transformers/exhibition-guides.ts

```diff
@@ -77,38 +87,35 @@ export function transformExhibitionGuideToExhibitionGuideBasic(
   }))(exhibitionGuide);
 }
 
-function transformRelatedExhibition(exhibition): Exhibit {
-  return {
-    exhibitType: 'exhibitions',
-    item: undefined,
-    id: exhibition.id,
-    title: (exhibition.data && asText(exhibition.data.title)) || '',
```

Comment:
Useful note: the `asTitle(…)` helper exists when you want to transform text and need to get *something* back.